data/wanderer/wanderers start.txt

```diff
@@ -2727,13 +2727,41 @@ mission "Wanderers Ap'arak 3: Pug"
 	source "Varu Tev'kei"
 	to offer
 		has "Wanderers Ap'arak 2: done"
-	npc kill
+	npc save
 		personality staying unconstrained uninterested
 		government "Pug (Wanderer)"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
+	to complete
+		never
+
+mission "Wanderers Ap'arak 3: Pug 2"
+	landing
+	invisible
+	source "Varu Tev'kei"
+	to offer
+		has "Wanderers Ap'arak 2: done"
+		not "Wanderers Ap'arak 3: done"
+	npc save
+		personality staying unconstrained uninterested
+		government "Pug (Wanderer)"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
-	on complete
-		dialog `Shortly after defeating the Pug ships, you receive a message from Sobari Tele'ek: "We are [aware? incredulous?] of reports that you have [exterminated, eradicated] our Keepers in the Ap'arak system. This is unacceptable! From now on, you are no longer welcome in our [territory, space]."`
+	to complete
+		never
+
+mission "Wanderers Pug Hostile"
+	priority
+	landing
+	invisible
+	deadline 1
+	source
+		government "Wanderer"
+	to offer
+		has "Wanderers Ap'arak 3: Pug: failed"
+		has "Wanderers Ap'arak 3: Pug 2: failed"
+	on offer
+		conversation
+			`Having defeated the Pug ships guarding Wanderer space, you receive a message from Sobari Tele'ek: "We are [aware? incredulous?] of reports that you have [exterminated, eradicated] our Keepers in the Ap'arak system. This is unacceptable! From now on, you are no longer welcome in our [territory, space]."`
+				launch
```

Comment:
The `launch` keyword is take off and accept. Use `flee` for take off and decline behavior, which removes the need for the deadline.